app/views/internal/configs/show.html.erb

```diff
@@ -11,16 +11,74 @@
   </h2>
 </div>
 
+<div class="row my-3" id="getStarted">
+  <div class="card w-100">
+    <div class="card-header" id="getStartedHeader" data-toggle="collapse"
+      data-target="#getStartedBodyContainer" aria-expanded="true" aria-controls="getStartedBodyContainer">
+      <h2 class="d-inline">Get Started</h2>
+      <button class="btn btn-secondary float-right" type="button">
+        Toggle Get Started
+      </button>
+    </div>
+
+    <div id="getStartedBodyContainer" class="collapse show hide p-3" aria-labelledby="siteConfigHeader">
+      <%= form_for(SiteConfig.new, url: internal_config_path) do |f| %>
+
+        <% VerifySetupCompleted::MANDATORY_CONFIGS.each do |config_key| %>
+          <!-- we need to list the config as seperate fields if the data structure is a Hash -->
+          <% if SiteConfig.send(config_key).is_a?(Hash) %>
```

Comment:
We have this if because we need to list the config as separate fields if the data structure is a Hash. We dont need to do this for an array or string. It feels weird to do this but without hardcoding the entire get started section I'm not sure what else we can do.

Reply:
You'd be surprised at how many `is_a?` checks can be found inside Rails itself, so don't feel too bad about this.